.github/workflows/stale.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow warns and then closes issues and PRs that have had no activity for a specified amount of time.
+#
+# You can adjust the behavior by modifying this file.
+# For more information, see:
+# https://github.com/actions/stale
+name: Mark stale issues and pull requests
+
+on:
+  schedule:
+    - cron: '00 03 * * *'
+  workflow_dispatch:
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+
+    steps:
+      - uses: actions/stale@v5
+        with:
+          # for now, we will only label PRs as stale, not issues
+          days-before-issue-stale: -1
+          days-before-pr-stale: 21
+          days-before-pr-close: 7
```

Comment:
Ok, I see this is the default but it's fine to include it for the action to be more descriptive.